VmAgent.Core/MultiLogger.cs

```diff
@@ -14,11 +14,22 @@ public class MultiLogger : ILogger
     {
         private readonly ILogger _logger;
         private readonly TelemetryClient _genevaTelemetryClient;
+        private ILogger _genevaOTelLogger;
+        private const string VmAgentLoggerName = "VmAgentLogger";
 
-        public MultiLogger(ILogger logger, TelemetryClient genevaTelemetryClient)
+        public MultiLogger(ILogger logger, TelemetryClient genevaTelemetryClient, ILogger genevaLogger = null)
         {
             _logger = logger ?? throw new ArgumentException("logger cannot be null");
+
+            // To enable both GenevaAgentChannel and OpenTelemetry, we have 2 ILogger parameters here.
+            // We will replace GenevaAgentchannel with Opentelemetry, once we verify all logs both in test and prod environments.
             _genevaTelemetryClient = genevaTelemetryClient ?? throw new ArgumentException("genevaTelemetryClient cannot be null");
+            _genevaOTelLogger = genevaLogger;
+        }
+
+        public void SetUpLogger(ILoggerFactory loggerFactory)
```

Comment:
why do we have the separate method if we're setting the same variable on the constructor?